pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -1047,6 +1048,27 @@ public void syncModeOptionMustBeUsed() {
     assertThat(commandErrorOutput.toString()).isEmpty();
   }
 
+
+  @Test
+  public void fastSyncTimeoutOptionMustBeUsed() {
+
+    parseCommand("--sync-mode", "FAST", "--fast-sync-max-wait-time", "30");
```

Comment:
You may try with values that are not the default values to be sure that they are taken in account and that it's not just the default.

Reply:
done